dashboard/src/components/AipDeletionRequestDialog.vue

```diff
@@ -41,42 +41,48 @@ const request = () => {
         <div class="modal-header">
           <h5 class="modal-title">Delete AIP</h5>
         </div>
-        <div class="modal-body">
-          <p>
-            This will initiate a <strong>deletion request</strong> for the
-            following AIP and any related replicas.
-          </p>
-          <p>
-            <strong>{{ aipStore.current?.name }}</strong>
-          </p>
-          <p>
-            You MUST provide a reason for the deletion. The request will be
-            reviewed before deletion.
-          </p>
-          <div>
-            <label for="reason" class="form-label"
-              >Reason for AIP deletion</label
+        <form @submit.prevent="request">
+          <div class="modal-body">
+            <p>
+              This will initiate a <strong>deletion request</strong> for the
+              following AIP and any related replicas.
+            </p>
+            <p>
+              <strong>{{ aipStore.current?.name }}</strong>
+            </p>
+            <p>
+              You MUST provide a reason for the AIP deletion. The request will
+              be reviewed before deletion.
+            </p>
+            <div>
+              <label for="reason" class="form-label">Reason:</label>
+              <textarea
+                class="form-control"
+                id="reason"
+                rows="3"
+                v-model="reason"
+                required
```

Comment:
This is cool, I didn't know they'd added client-side validation to the HTML spec. :+1: